menu/menu_displaylist.c

```diff
@@ -985,6 +989,96 @@ static unsigned menu_displaylist_parse_core_manager_list(
    return count;
 }
 
+#ifdef HAVE_MIST
+static unsigned menu_displaylist_core_manager_steam_list(
```

Comment:
For consistency, please name this `menu_displaylist_parse_core_manager_steam_list()`